micall/core/plot_contigs.py

```diff
@@ -80,17 +80,24 @@ def __init__(self, a, b, ys):
         height = 10
         color = 'red'
         opacity = '1.0'
-        super(SmoothCoverage, self).__init__(a, b, ys, height, color, opacity)
+        super(ConcordanceLine, self).__init__(a, b, ys, height, color, opacity)
```

Comment:
This is correct, but no longer required in Python 3. You can use `super()` without arguments.